examples/dt/unified/delta/kustomization.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+components:
+  - ../../../../../dt/unified/delta/
```

Comment:
`  - ../../../../dt/unified/delta/`

I think there's one too many `..`

Reply:
Yes, Thank you for pointing it out. 

Let me fix it in the next commit.